templates/html.mustache

```diff
@@ -297,18 +297,20 @@
     {{/file_has_extensions}}
     {{#file_services}}
     <h3 id="{{service_full_name}}">{{service_name}}</h3>
-    {{#p}}{{service_description}}{{/p}}
+    {{#p}}{{{service_description}}}{{/p}}
```

Comment:
The triple tells it to render unescaped HTML so it will let us use the html formatting